p_sets/2/caesar.rb

```diff
@@ -0,0 +1,42 @@
+# HAIL CAESAR
+# program that encrypts messages using Caesars cipher.
+# program must accept a single command-line argument,k, a non-negative integer.
+# Return 1 if the CLA doesnt meet these criteria
+# Otherwise prompt the user for a string of plaintext and then output ciphertex.
+# non-alphabetical characters should be outputted unchanged.
+# After outputting this ciphertext, your program should exit, returning 0.
+# you may not assume that k will be less than or equal to 26
+#  should work for all non-negative ints less than 2v31 - 26
+require_relative 'encryption'
+class Caesar
+  include Encryption
+
+  def make_key(integer)
```

Comment:
Also, to pick up Hywel's point about argument names, `integer` is possible a better name than `x` would be, but still can be improved. To use this class I'd need to know what the integer represents.

In this case, it looks like the user will be passing in a number that they intend to use as a key, and you're constraining it to the range `0..25`

The most explicit you could be in this case is probably: `def valid_key(intended_key)`.

You could shorten that either to `def key(intended_key)` or `def valid_key(key)`, but `def key(key)` would remove too much context I think.